service/src/main/java/bio/terra/tanagra/plugin/PluggableConfiguration.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.tanagra.plugin;
+
+import java.util.Map;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "tanagra.pluggable")
```

Comment:
This `name` + `parameters` structure is fine, but I think it should go in the underlay config so we can vary it per-dataset, not the Spring application config which would be per-deployment. Recommend putting it in the top-level `UFUnderlay` [class](https://github.com/DataBiosphere/tanagra/blob/main/underlay/src/main/java/bio/terra/tanagra/serialization/UFUnderlay.java) or add a sub-class for just the plugins.

Reply:
I'll move this to the underlay

Reply:
Done.